pull_into_place/commands/01_setup_workspace.py

```diff
@@ -252,6 +252,21 @@ def install(workspace, flags_path):
         else:
             scripting.touch(workspace.flags_path)
 
+class FragmentWeights:
+    prompt = None
+    description = """\
+Fragment weights files, describing how any fragment-based filter should
+behave. A simple and standard set of weights are provided."""
+
+    @staticmethod
+    def install(workspace):
+        simple_weights = pipeline.big_job_path('simple_fragment.wts')
+        standard_weights = pipeline.big_job_path('standard_fragment.wts')
+        shutil.copyfile(simple_weights,
+                workspace.simple_fragment_weights_path)
+        shutil.copyfile(standard_weights,
+                workspace.standard_fragment_weights_path)
+
```

Comment:
I agree, and I always end up copying files over from other workspaces anyway. What might be good, in addition to just having everything get copied over automatically, would be to have an "inputs" directory where everything Rosetta uses goes (input file, flags, scripts, etc.). That way, the user can clutter up the root directory as much as they want, and when they go to create a new workspace, they can just copy everything in the "inputs" directory. If nothing else, it would fit my workflow really well!